acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/Accounts.java

```diff
@@ -42,15 +36,15 @@ public Account richBenefactor() {
     return benefactor;
   }
 
-  public BigInteger balance(final Account account) {
```

Comment:
Don't bleed the exception through this layer (it's inconsistent with Pantheon)

Reply:
Done! This is the reason that the IOExceptions were needed!!!!